packages/merge-fmt/merge-fmt.0.3/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+maintainer:   "hugo.heuzard@gmail.com"
+authors:      "Hugo Heuzard"
+homepage:     "https://github.com/hhugo/merge-fmt"
+doc:          "https://hhugo.github.io/merge-fmt/"
+bug-reports:  "https://github.com/hhugo/merge-fmt/issues"
+dev-repo:     "git+https://github.com/hhugo/merge-fmt.git"
+license:      "MIT"
+
+build: [["dune" "build" "-p" name "-j" jobs]]
```

Comment:
Reading the source code I'm noticing that there are some tests. Do you want to run them?
```suggestion
build: [["dune" "build" "-p" name "-j" jobs]]
run-test: [["dune" "runtest" "-p" name "-j" jobs]]
```

Reply:
I don't think so, they are too fragile, they depend on `git` output.

Reply:
ok fair enough